src/lib/m2/index.js

```diff
@@ -116,5 +128,29 @@ export default new r.Struct({
   }),
   unknown2: new r.Optional(r.uint32le, function() {
     return this.flags === 8;
-  })
+  }),
+
+  isAnimated: function() {
+    let animated = false;
+
+    this.bones.forEach((bone) => {
+      if (bone.isAnimated) {
+        animated = true;
+      }
+    });
+
+    this.transparencies.forEach((transparency) => {
+      if (transparency.isAnimated) {
+        animated = true;
+      }
+    });
+
+    this.colors.forEach((color) => {
+      if (color.color.isAnimated || color.alpha.isAnimated) {
+        animated = true;
+      }
+    });
+
+    return animated;
+  }
```

Comment:
Gotcha, I'll have to check that out. And yeah... probably unnecessary at this point.

Maybe we could get something set up in the test suites for Blizzardry to try benchmark load performance for a specific, complicated M2 (like Illidan)? For example: load Illidan from scratch 10 times in a row, average the load time, report it back on the screen. Then we'd at least have an easier way to compare with previous results on the same systems when trying to optimize code paths running during parsing.